src/System/System.php

```diff
@@ -319,6 +319,43 @@ public static function getCPUUtilisation(int $id = 0): int
         }
     }
 
+    /**
+     * Get percentage CPU usage (between 0 and 100)
+     * Refference for formula: https://stackoverflow.com/a/23376195/17300412
+     *
+     * @param  int  $duration
+     * @return float
+     *
+     * @throws Exception
+     */
+    public static function getCPUUsage(int $duration = 1): float
```

Comment:
How does this compare to `getCPUUtilisation`? 

Also, at some point we might need to have OS level adapters instead of multiple switch-cases.

Reply:
`getCPUUtilisation` could not be used to get percentage CPU usage, that's what I attempted initially. Absolute CPU usage requires delta time, meaning the check runs 2 times and calculates the difference in usage.

This method follows formula suggested to calculate CPU usage specifically.

Reply:
We could not find a logical use case for this method. I am removing it. (getCPUUtilisation)